src/arch/riscv/kernel/vspace.c

```diff
@@ -180,8 +180,8 @@ BOOT_CODE void map_it_pt_cap(cap_t vspace_cap, cap_t pt_cap)
     *targetSlot = pte_new(
                       (addrFromPPtr(pt) >> seL4_PageBits),
                       0, /* sw */
-                      1, /* dirty */
-                      1, /* accessed */
+                      0, /* dirty */
```

Comment:
```suggestion
                      0, /* reserved */
```
I think the comment should be update to make clear these bit mean something different now